src/requirements.txt

```diff
@@ -0,0 +1,4 @@
+Django>=2.1.0,<2.2.0
```

Comment:
Good practice is to fix the version such that your environment has a determined version no matter when you run/deploy it.

Reply:
Changed!